Sources/LiveKit/Track/VideoTrack.swift

```diff
@@ -20,34 +20,42 @@ public protocol VideoTrack: Track {
 
 }
 
+@objc public protocol VideoRenderer: RTCVideoRenderer {
+    /// Whether this ``VideoRenderer`` should be considered visible or not for AdaptiveStream.
+    /// This will be invoked on the .main thread.
+    var adaptiveStreamIsEnabled: Bool { get }
+    /// The size used for AdaptiveStream computation. Return .zero if size is unknown yet.
+    /// This will be invoked on the .main thread.
+    var adaptiveStreamSize: CGSize { get }
+}
```

Comment:
Now any object conforming to this protocol can be added to `VideoTrack.add(videoRenderer:)`.